### PR TITLE
Updated ‘Rainbow’ dependency

### PR DESCRIPTION
**Pull request by regexident**

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -8,8 +8,7 @@ let package = Package(
         .package(url: "https://github.com/JohnSundell/Files.git", from: "2.0.0"),
         .package(url: "https://github.com/daniel-pedersen/SKQueue.git", from: "1.1.0"),
         .package(url: "https://github.com/surfandneptune/CommandCougar.git", from: "1.0.0"),
-//        .package(url: "https://github.com/onevcat/Rainbow", from: "3.0.0"),
-        .package(url: "https://github.com/regexident/Rainbow", .branch("manifest")),
+        .package(url: "https://github.com/onevcat/Rainbow", from: "3.0.0"),
```
Comment (BenchR267) [inline Package.swift:11]: Should this be pinned to `from: 3.0.3` if this is the version that contains the necessary fix?

Comment (BenchR267) [reply Package.swift:11]: @regexident? 😆

Comment (regexident) [reply Package.swift:11]: `Package.resolved` pins it to `3.0.3` and `swift package resolve` should grab `3.0.3` over `3.0.0`, no?

Comment (BenchR267) [reply Package.swift:11]: If `3.0.3` would be removed `swift package resolve` would use 3.0.2 instead of showing an error

Comment (regexident) [reply Package.swift:11]: Fair point, fixed.

**Pull request merged**